kube/minikube/deployments/regression-tracker-deployment.yaml

```diff
@@ -0,0 +1,54 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2023 KernelCI Authors.
+# Author: Sanskar Bhushan <sbdtu5498@gmail.com>
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kernelci-pipeline-regression-tracker
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kernelci-pipeline-regression-tracker
+  template:
+    metadata:
+      labels:
+        app: kernelci-pipeline-regression-tracker
+    spec:
+      containers:
+        - name: kernelci-pipeline-regression-tracker
+          image: kernelci/staging-kernelci
+          env:
+            - name: KCI_API_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: kernelci-pipeline-secrets
+                  key: kci-api-token
+            - name: SETTINGS
+              valueFrom:
+                configMapKeyRef:
+                  name: kernelci-pipeline-config
+                  key: settings
+          command:
+            - "/usr/bin/env"
+            - "python3"
+            - "/home/kernelci/pipeline/regression_tracker.py"
+            - "--settings=$(SETTINGS)"
+            - "run"
+            - "--api-config=k8s-host"
+          volumeMounts:
+            - name: pipeline-volume
+              mountPath: /home/kernelci/pipeline
+            - name: config-volume
+              mountPath: /home/kernelci/config
+          # Resource limits needs to be discussed.
```

Comment:
I guess it's up to the user to decide, maybe some minimal values can be provided here as an example?

Reply:
Not sure if it has some default values. I'll check the documentation.

Reply:
There are no default values for resource limits if not provided.
We need to analyze the resource usage of different containers before adding limits here.

Reply:
Well I don't really think we can do that, it just depends on the scale of the deployment.  Having something very simple like 0.5 CPU and 256 MB of memory would probably be fine, then users can adjust it as needed?

Reply:
...as limits, and maybe half of that as requirements?

Reply:
Yes, we can do that.